src/Kiosk.Api/Services/MajorsService.cs

```diff
@@ -56,4 +59,84 @@ public async Task<IEnumerable<MajorResponse>> GetTranslatedMajors(FindMajorsRequ
     
         return majorsList.Select(major => MapTranslatedMajor(major, findMajorsRequest.Language));   
     }
+
+    public async Task CreateMajors(IEnumerable<CreateMajorRequest> createMajorsRequest,
```

Comment:
Shouldn't we add any validation to check if Major doesn't exist?

Reply:
it would have to check if degree and name are not existing in db, I think i will add it in another task because it is not too important for now